app/code/Magento/Variable/Model/Variable.php

```diff
@@ -162,4 +179,39 @@ public function getVariablesOptionArray($withGroup = false)
         }
         return $variables;
     }
+
+    /**
+     * Check if the given store (or the current one) has the given
+     * variable set with the given text value
+     *
+     * @param string $variableCode
+     * @param string $expectedValue
+     * @param string $type
+     * @param int $storeId
+     *
+     * @return boolean
+     */
+    public function customVariableHasValue(
```

Comment:
Indeed, lack of focus for me. Very strange that PHP Storm didn't rise a warning.